### PR TITLE
AGI: Add pirated version of GoldRush! to detection table

### PR DESCRIPTION
**Pull request by ZvikaZ**
There are 2 bug reports regarding one version of GoldRush that
wasn't recognized by the detection table.

On bug #3320, it was explained to be a pirated version. Therefore,
this update, in order to avoid further reporting of the same issue.

It closes bug #11308.

### CODE REVIEW

File: engines/agi/detection_tables.h [hunk 1]
```diff
@@ -256,6 +273,9 @@ static const AGIGameDescription gameDescriptions[] = {
 	// Gold Rush! (PC 3.5") 2.01 12/22/88 [AGI 3.002.149]
 	GAME3("goldrush", "2.01 1988-12-22 3.5\"", "grdir", "6a285235745f69b4b421403659497216", 0x3149, GID_GOLDRUSH),
 
+    // Gold Rush!  2.01 12/22/88 - pirated copy, according to https://bugs.scummvm.org/ticket/3220
```
Comment (mgerhardy) [inline engines/agi/detection_tables.h]: This is a whitespace error. Tabs should be used.

**Comment by lotharsm** [comment]
I wonder if adding pirated versions to the detection tables is even allowed...

**Comment by criezy** [comment]
@lotharsm Yes, we have the `ADGF_PIRATED` flag specially for that.
Adding this seems like a good idea to me.

**Comment by lotharsm** [comment]
> 
> 
> @lotharsm Yes, we have the `ADGF_PIRATED` flag specially for that.
> Adding this seems like a good idea to me.

Alright, thanks a lot for the feedback! Looks good to me too then.

**Comment by bluegr** [comment]
Thanks for your work! Merging

**Pull request merged**